Documentation/ApiOverview/Autoloading/Index.rst

```diff
@@ -13,104 +14,81 @@ and :ref:`XCLASS <xclasses>` handling.
 
 As a developer you should always instantiate classes either through
 :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()` or with the Extbase
-:ref:`ObjectManager <t3cmsapi:TYPO3\\CMS\\Extbase\\Object\\ObjectManager>`,
-which internally uses :php:`makeInstance()` again.
-
-.. important::
-
-   Since TYPO3 CMS 6.0 and the introduction of namespaces, developers are strongly encouraged
-   to use the namespaces. When using namespaces it is not necessary to explicitely declare
-   classes in an autoloader files. All namespaced classes are found and loaded automatically.
-
-   Please refer to the :ref:`namespaces` chapter for more information.
-
-
-.. _autoload-autoloader:
-
-Naming convention or autoloader file
-====================================
-
-In TYPO3 every class must reside in its own file, i.e. there should be only
-one class per PHP file. Extensions must not use :php:`require()` or :php:`include()`
-*to load class files*, but instead use the TYPO3 core API to automatically require a file
-upon request of the class.
-
-A developer has two options to help the core find a specific class:
-
-- Use the class naming convention and file location.
-
-- Register a class name together with its location in an :php:`ext_autoload.php` file.
-
-If it's not possible to stick to the class naming and file location conventions - for whatever
-reasons - or if you don't want to use namespaces, you can add a file to your extension called
-:file:`ext_autoload.php` in the base directory. Its task is to inform the autoloader about the
-location of each class file. The autoloader automatically searches for this file when a class is
-requested.
-
-The :file:`ext_autoload.php` file simply has to return a one-dimensional array
-with the class name as key and the file location as value. No other code is allowed in this file.
+:ref:`ObjectManager <t3cmsapi:TYPO3\\CMS\\Extbase\\Object\\ObjectManager>`
+(which internally uses :php:`makeInstance()` again).
 
+.. _autoloading_since_typo3_7:
+Autoloading classes since TYPO3 7.x
+===================================
 
-.. _autoload-examples:
+TYPO3 6.2 was still delivered with a couple of different autoloaders, that all had different approaches and rules to find a class. This led to the naming conventions in and outside extbase and the optional `ext_autoload.php` file to load classes that didn't follow the conventions. Since TYPO3 7.0 all this is gone and there is only a single autoloader left, the one of composer. No matter if you run TYPO3 in composer mode or not, TYPO3 uses the composer autoloader to resolve all class file locations. However, the autoloader is little bit more sophisticated in composer mode as it then supports `PSR-4` autoloading.
```

Comment:
That's why I wrote, that #149 should be merged before. Let's focus on that PR and simply use the same version for latest then.